example/deployment-rbac.yaml

```diff
@@ -39,6 +39,12 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8080
+          initialDelaySeconds: 15
+          timeoutSeconds: 3
```

Comment:
You could add 
```yaml
          ports:
          - name: readyz 
            port: 8080
```
That way the Dashboard can display the port like that:
![screen shot 2018-08-08 at 1 51 04 pm](https://user-images.githubusercontent.com/5884148/43854883-2169dd6e-9b12-11e8-85dd-1e6834b97bef.png)

Reply:
Weird, couldn't find it in my dashboard on minikube but a good idea so updated the example to use a named port.